src/shared/components/comment-node.tsx

```diff
@@ -667,9 +689,12 @@ export class CommentNode extends Component<CommentNodeProps, CommentNodeState> {
         {this.state.showBanDialog && (
           <form onSubmit={linkEvent(this, this.handleModBanBothSubmit)}>
             <div class="form-group row">
-              <label class="col-form-label">{i18n.t('reason')}</label>
+              <label class="col-form-label" htmlFor="mod-ban-reason">
+                {i18n.t('reason')}
+              </label>
               <input
                 type="text"
+                id="mod-ban-reason"
```

Comment:
These will probably need to use the random string generated above, because there can be many comment nodes and many ban forms open up in one page.

Reply:
Yep good call, I will add the id at the end of the string here